packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -95,8 +153,9 @@ export const SetupFlopFlipProvider = (props: Props) => {
   }
 
   return (
-    <ConfigureFlopFlip<typeof ldAdapter>
-      adapter={ldAdapter}
+    <ConfigureFlopFlip<typeof combineAdapters>
+      adapter={combineAdapters}
+      // @ts-expect-error
```

Comment:
Will have to look into it. Typing flopflip has become rocket science recently when combining adapters :)

Reply:
What's the error you get?

Reply:
Right now, the document node is new as the fetch expects a string.

```
packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx:161:7 - error TS2322: Type '{ user: { key: string | undefined; }; memory: { user: { key: string | undefined; }; }; launchdarkly: { sdk: { clientSideId: string; }; flags: {}; user: { key: string | undefined; custom: CustomLDUser; }; }; graphql: { ...; }; }' is not assignable to type 'TCombinedAdapterArgs'.
  Type '{ user: { key: string | undefined; }; memory: { user: { key: string | undefined; }; }; launchdarkly: { sdk: { clientSideId: string; }; flags: {}; user: { key: string | undefined; custom: CustomLDUser; }; }; graphql: { ...; }; }' is not assignable to type '{ launchdarkly?: TLaunchDarklyAdapterArgs | undefined; localstorage?: TLocalStorageAdapterArgs | undefined; memory?: TBaseAdapterArgs<...> | undefined; splitio?: TSplitioAdapterArgs | undefined; graphql?: TGraphQLAdapterArgs | undefined; }'.
    Types of property 'graphql' are incompatible.
      Type '{ user: { key: string | undefined; }; uri: string; query: DocumentNode; pollingInteralMs: number; getRequestHeaders: (props: Props) => TGraphQLAdapterRequestHeaders; parseFlags: (fetchedFlags: TFetchedFlags) => { ...; }; cacheIdentifier: string; }' is not assignable to type 'TGraphQLAdapterArgs'.
        Type '{ user: { key: string | undefined; }; uri: string; query: DocumentNode; pollingInteralMs: number; getRequestHeaders: (props: Props) => TGraphQLAdapterRequestHeaders; parseFlags: (fetchedFlags: TFetchedFlags) => { ...; }; cacheIdentifier: string; }' is not assignable to type '{ fetcher?: ((input: RequestInfo, init?: RequestInit | undefined) => Promise<Response>) | undefined; uri: string; query: string; pollingInteralMs?: number | undefined; getQueryVariables?: ((adapterArgs: TGraphQLAdapterArgs) => unknown) | undefined; getRequestHeaders?: ((adapterArgs: TGraphQLAdapterArgs) => Record<.....'.
          Types of property 'query' are incompatible.
            Type 'DocumentNode' is not assignable to type 'string'.

161       adapterArgs={adapterArgs}
          ~~~~~~~~~~~

  node_modules/@flopflip/types/dist/declarations/src/index.d.ts:186:5
    186     adapterArgs: TConfigureAdapterArgs<TAdapterInstance>;
            ~~~~~~~~~~~
    The expected type comes from property 'adapterArgs' which is declared here on type 'IntrinsicAttributes & Pick<Props<CombineAdapters>, "children" | "adapter" | "adapterArgs"> & Partial<Pick<Props<CombineAdapters>, "shouldDeferAdapterConfiguration" | "defaultFlags">> & Partial<...>'
```